### DISCUSSION TITLE
Dropdown component is cut-off in the modal

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#1384] [Q&A] [answered]
Hi team,

we're trying to implement a combobox component in the modal like this. (This will be a long list with 100+ items.)

<img width="712" alt="Screen Shot 2021-04-26 at 3 49 51 PM" src="https://user-images.githubusercontent.com/56697620/116155264-1fe85080-a6a7-11eb-96d4-55c76883bec4.png">

However, when we implemented it, the combobox gets cut-off in the modal, and adds a scrollbar in the modal instead of inside a combobox.

<img width="623" alt="Screen Shot 2021-04-26 at 3 48 13 PM" src="https://user-images.githubusercontent.com/56697620/116155127-ea436780-a6a6-11eb-9430-ce7fdc60639b.png">

Is there any way to keep the combobox outside of the modal (as in the first mock) and add a scrollbar in the combobox, not in the modal?

Thanks!

**Comment by zahnster** [comment, marked as answer]
Hi @charleneyjh1

We've had a similar issue to yours come up before in GitHub Discussions: https://github.com/twilio-labs/paste/discussions/746

Please review SiTaggart's comment in that topic for advice on your situation.
